scripts/build.sh

```diff
@@ -166,7 +166,7 @@ tvos_flags=(
   -destination 'platform=tvOS Simulator,name=Apple TV'
 )
 watchos_flags=(
-  -destination 'platform=WatchOS Simulator,name=Apple Watch Series 7 - 45mm'
+  -destination 'platform=WatchOS Simulator,name=Apple Watch Series 7 (45mm)'
```

Comment:
very sneaky but it looks like it's also now "**w**atchOS Simulator". 
```diff
- platform=WatchOS Simulator
+ platform=watchOS Simulator
```

Reply:
Fixed. It does seem case-insensitive though since the change was sufficient to start passing the spm watchOS tests